### ISSUE TITLE
When Database Shells Configuration Provider is used there is an exception on startup

### ISSUE THREAD

**Issue by MikeAlhayek** [OrchardCMS/OrchardCore#14496]
@jtkech When [Database Shells Configuration Provider](https://docs.orchardcore.net/en/latest/docs/reference/core/Shells/#database-shells-configuration-provider) is used there is an exception on startup:

```
System.InvalidOperationException: StreamConfigurationProviders cannot be loaded more than once.
   at Microsoft.Extensions.Configuration.StreamConfigurationProvider.Load()
   at Microsoft.Extensions.Configuration.ConfigurationRoot.Reload()
   at OrchardCore.Environment.Shell.ShellSettingsManager.LoadSettingsAsync(String tenant)
   at OrchardCore.Environment.Shell.Distributed.DistributedShellHostedService.LoadingAsync()
   at OrchardCore.Environment.Shell.ShellHost.PreCreateAndRegisterShellsAsync()
   at OrchardCore.Environment.Shell.ShellHost.InitializeAsync()
   at OrchardCore.Modules.ModularTenantContainerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
```

_Originally posted by @wAsnk in https://github.com/OrchardCMS/OrchardCore/issues/14117#issuecomment-1759957496_

**Comment by jtkech** [comment]
Okay, we try to reload to prevent to rebuild some configs, I will fix it this night.

**Comment by jtkech** [comment]
Okay, fixed by #14499
